source/agora/consensus/EnrollmentManager.d

```diff
@@ -711,6 +668,26 @@ public class EnrollmentManager
         return this.cycle_preimages[height];
     }
 
+    /***************************************************************************
+
+        Create signature noise for enrollment data
+
```

Comment:
Please add a description explaining what it is, why we need it, why we derive it, how, etc...